Drop the -git<date> suffix from the pkg-config version on non-Windows builds, which abuild rejects

--- a/src/meson.build
+++ b/src/meson.build
@@ -412,11 +412,12 @@
 else
     import('pkgconfig').generate(
             libpistache,
             name: 'Pistache',
             description: 'An elegant C++ REST framework',
             url: 'https://pistacheio.github.io/pistache/',
-            version: '@0@-git@1@'.format(version_str, version_git_date),
+            # abuild rejects a pkg-config version with a -git<date> suffix
+            version: version_str,
             filebase: 'libpistache',
             extra_cflags: public_args
     )
 endif
